platform/podjs/src/browserPod.ts

```diff
@@ -534,7 +540,7 @@ function endpointErrorMessage(fetchType: string, errorlog: string): string {
 }
 
 /**
- * @class BrowserEndpoint
+ * @class BrowserEndpoint @implements [[Endpoint]]
```

Comment:
[typedoc apparently does not have that tag either](https://typedoc.org/tags/internal/)